packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/ResultOptionsBar/index.js

```diff
@@ -35,42 +35,48 @@ export class ResultOptionsBar extends Component {
     } = this.props
 
     return (
-      <Menu attached="top" style={{ position: 'relative' }} tabular>
-        {itemEnabled && (
+      <Menu
+        attached="top"
+        className="icon"
+        style={{ position: 'relative' }}
+        tabular
+      >
+        <Menu.Item
+          active={tableActive}
+          name="table"
+          onClick={event => handleListTabClick(event)}
+        >
+          <Icon name="table" />
+        </Menu.Item>
+
+        {treeEnabled && (
           <Menu.Item
-            active={createItemActive || editItemActive}
+            active={treeActive}
             name="form"
-            onClick={event => handleFormTabClick(event)}
+            onClick={event => handleTreeTabClick(event)}
           >
-            <Icon name="wordpress forms" />
+            <Icon name="sitemap" />
           </Menu.Item>
         )}
 
-        {treeEnabled && (
+        {itemEnabled && (
           <Menu.Item
-            active={treeActive}
+            active={createItemActive || editItemActive}
             name="form"
-            onClick={event => handleTreeTabClick(event)}
+            onClick={event => handleFormTabClick(event)}
           >
-            <Icon name="sitemap" />
+            <Icon name="wordpress forms" />
           </Menu.Item>
         )}
 
-        <Menu.Item
-          active={tableActive}
-          name="table"
-          onClick={event => handleListTabClick(event)}
-        >
-          <Icon name="table" />
-        </Menu.Item>
         <Menu.Menu position="right">
           <Menu.Item>
             <Icon
+              className="icon ui secondary"
```

Comment:
remove, should be on Menu

Reply:
Talked to Lisa, this need to put in Menu.Menu.